File: eizaguirre9.java
import java.util.Scanner;
import java.util.Random;

public class eizaguirre9
{
	public static void main(String[]args)
	{
		Random rand = new Random();
		Scanner keyboard = new Scanner(System.in);

		int x = rand.nextInt(19) + 1;
		int y;//guess
	while(true)
	{
		do
		{
			System.out.println("Enter a number in between 1 and 19:");
			y = keyboard.nextInt();
			
			if(y < 1 || y > 19)
			{
				 System.out.println("Wrong number, try again. ");
			}
		}
		while (y <1 || y > 19);
		
		if(y == x)
		{
			for(int i = 0; i< y; i++)
			{
				 System.out.println("WINNER WINNER CHICKEN Dinner");
			}
			break;
		}
		else
		{ 
			System.out.println("you lost");

			if (y > x)
			{ 
				System.out.println("Your number is higher by" + (y-x));
			}
			else
			{
				 System.out.println("Your number is lower by" + (x-y));
			}
		}
	}
    }
}

File: eizaguirre9.c
/*Name: Esthepanie Izaguirre
* CSC 321
* Lab 09 */
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main()
{
        int y, z; //random and guess number
        srand(time (0));
        y = rand() % 19 + 1;
while(1)
{
do
{
 printf("Guess a number between 1  and 19: \n");
  scanf("%d", & z);
  if(z < 1 || z > 19)
  {
         printf("Wrong number");
  }
}
  while (z < 1 || z >19 );

  if(z == y)
  {
          for(int i = 0; i < z; i++)
          {
                  printf("WINNER WINNER Chicken Dinner\n");
          }
         break;
  }
  else
  {
        printf("You Lost!\n");
       if(z > y)
       {
       printf("Your estimate was lowe by %d\n",z - y );
       } else {
         printf("Your estimate was lowe by %d\n",y - z );
        }
    }
}
return 0;
}
